<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BMDialog</class>
 <widget class="QDialog" name="BMDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>283</width>
    <height>227</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BMDialog</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>SAD window size</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>pre filtering size</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>pre filtering cap</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>min. disparity</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>nr od disparities</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>uniqueness ratio</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>texture threshold</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QSpinBox" name="dispBox">
     <property name="minimum">
      <number>16</number>
     </property>
     <property name="maximum">
      <number>128</number>
     </property>
     <property name="singleStep">
      <number>16</number>
     </property>
     <property name="value">
      <number>64</number>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSpinBox" name="sadBox">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>31</number>
     </property>
     <property name="singleStep">
      <number>2</number>
     </property>
     <property name="value">
      <number>21</number>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="preSizeBox">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>21</number>
     </property>
     <property name="singleStep">
      <number>2</number>
     </property>
     <property name="value">
      <number>7</number>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="preCapBox">
     <property name="minimum">
      <number>3</number>
     </property>
     <property name="maximum">
      <number>51</number>
     </property>
     <property name="singleStep">
      <number>2</number>
     </property>
     <property name="value">
      <number>21</number>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSpinBox" name="minDispBox">
     <property name="minimum">
      <number>-64</number>
     </property>
     <property name="maximum">
      <number>64</number>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QSpinBox" name="uniqueBox">
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>21</number>
     </property>
     <property name="singleStep">
      <number>2</number>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QSpinBox" name="textureBox">
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>20</number>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
